### DISCUSSION TITLE
修复使用部分js框架时网页上的电子邮件地址显示[email protected]的问题 - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by Muska-Ami** [Muska-Ami/muska-ami-blog#42] [General]
https://blog.1l1.icu/2024/01/27/xiu-fu-shi-yong-bu-fen-js-kuang-jia-shi-wang-ye-shang-de-dian-zi-you-jian-di-zhi-xian-shi-email-protected-de-wen-ti/ 

 Keep doing, keep loving❤.

**Comment by ltzXiaoYanMo** [comment]
如何快速成为CSGO玩家？

1.把F键扣下来扔楼下

2.把之前的FPS游戏全忘了

3.遇到背对你的敌人，刀了

4.开箱拿蓝很赚，以上的卖了

5.对面第一绝对是挂

（？）

**Comment by Muska-Ami** [comment]
> @ltzXiaoYanMo
> 如何快速成为CSGO玩家？
> 
> 1.把F键扣下来扔楼下
> 
> 2.把之前的FPS游戏全忘了
> 
> 3.遇到背对你的敌人，刀了
> 
> 4.开箱拿蓝很赚，以上的卖了
> 
> 5.对面第一绝对是挂
> 
> （？）

byd水评论能不能不要无中生草啊（
